core_domain.process_model_execution/ioc_module.js

```diff
@@ -139,6 +139,7 @@ function registerFlowNodeHandlers(container) {
         container.get('CoreDomain.ProcessDefinitionRepository'),
         container.get('CoreDomain.ProcessInstanceRepository'),
         container.get('ResumeProcessService'),
+        container.get('ProcessInstanceManagementService'),
```

Comment:
Bitte alphabetisch sortieren

```suggestion
        container.get('ProcessInstanceManagementService'),
        container.get('ResumeProcessService'),
```